src/data_hub/plotting.py

```diff
@@ -0,0 +1,126 @@
+"""module to make plots from a Deep Origin dataframe"""
+
+from beartype import beartype
+from bokeh.io import show
+from bokeh.layouts import column, row
+from bokeh.models import ColumnDataSource, CustomJS, Select
+from bokeh.plotting import figure
+from deeporigin.data_hub.dataframe import DataFrame
+
+
+@beartype
+def create_interactive_scatter(df: DataFrame):
+    """function to make a scatter plot from a Deep Origin dataframe"""
+
+    figure_width = 500
+    select_width = int(figure_width * 0.3)
+
+    if df.shape[1] < 2:
+        raise ValueError("DataFrame must contain at least two columns.")
+
+    cols = df.attrs["metadata"]["cols"]
+    cols = [col["name"] for col in cols if col["type"] in ["float", "integer"]]
```

Comment:
Reuse the numeric column type list you declared elsewhere?

Reply:
Sadly that's it tests

Reply:
Move it? 😃